packages/tools/src/tools/segmentation/strategies/fillSphere.ts

```diff
@@ -26,37 +27,44 @@ function fillSphere(
     segmentIndex,
     segmentationId,
     points,
+    lazyCalculation,
   } = operationData;
-
   const { imageData, dimensions } = segmentation;
   const scalarData = segmentation.getScalarData();
-  const scalarIndex = [];
+  const modifiedSlicesToUse = new Set<number>();
 
-  const callback = ({ index, value }) => {
-    if (segmentsLocked.includes(value)) {
-      return;
+  let pointsChunks;
+  if (lazyCalculation) {
```

Comment:
Please add a comment describing the difference between the lazy and non-lazy calculation. Perhaps even go so far as to describe the difference in `OperationData.points` between lazy and non-lazy.